source/GreenEnergyHub.Charges/source/GreenEnergyHub.Charges.IntegrationTests/TestHelpers/MessageHubHelper.cs

```diff
@@ -31,5 +32,22 @@ public static async Task AssertPeekReceivesReplyAsync(this MessageHubSimulation
             // Invokes the domain and ensures that a reply to the peek request is received
             await messageHub.PeekAsync(); // Throws if corresponding peek reply is not received
         }
+
+        /// <summary>
+        /// Initiates a peek and assert that a reply is received.
+        /// </summary>
+        public static async Task AssertPeekReceivesReplyAsync(this MessageHubSimulation messageHub, string correlationId, int noOfMessageTypes)
```

Comment:
Perhaps like this and remove the original method?
```suggestion
        public static async Task AssertPeekReceivesReplyAsync(this MessageHubSimulation messageHub, string correlationId, int noOfMessageTypes = 1)
```

Reply:
Simplified so only one method exist as suggested